README.md

```diff
@@ -7,22 +7,47 @@ npm install vue-lazzy-image
 ## How use lib
 
 ```ecmascript 6
-import lazyload from 'vue-lazzy-image'
+import CoreLazyLoading from 'vue-lazzy-image'
```

Comment:
I don't think that CoreLazyLoading could be a better name.  You could do something like this:
```javascript
import ImageLazy from 'vue-lazzy-image/@core'
```